docs/src/Flex.doc.js

```diff
@@ -135,26 +135,6 @@ card(
   />,
 );
 
-card(
```

Comment:
@dangerismycat could you rebase? With Next.js, the docs pages live under `pages/`